fabfile.py

```diff
@@ -2,25 +2,29 @@
 
 from fabric.api import *
 import sys
+import os
 
-# USAGE: $ fab -R pub get_ut {input_file} {output_file}
+# USAGE: $ fab -R pub get_ut:input={input_file},output={output_file}
 
 env.roledefs = {
     'pub': ['openapc@pub'],
+    'test': ['bup@pub'],
 }
 
 def prepare():
     local("git pull origin master")
 
-def get_ut(input_file,output_file):
-    if input_file == '' or output_file == '':
+def get_ut(input,output):
+    if input == '' or output == '':
         sys.exit("Input file and output file are required.")
 
     put("bin/fetch.pl", "openapc/fetch.pl")
-    put("data/" + input_file, "openapc/" + input_file)
+    in_file = os.path.basename(input)
+    put(input, "openapc/" + in_file)
 
     with cd("openapc/"):
-        run("perl fetch.pl --input " + input_file + " --output tmp_ut.csv")
+        print(blue("Running script!"))
+        run("perl fetch.pl --input " + in_file + " --output tmp_ut.csv")
         get("tmp_ut.csv", "tmp_ut.csv")
-        local("R CMD BATCH R/isi_add.r tmp.txt " + output_file)
+        local("R CMD BATCH '--args tmp.txt " + output +"' R/isi_add.r")
```

Comment:
Does not work yet:
`[localhost] local: R CMD BATCH '--args tmp.txt python/out_ut.csv' R/isi_add.r`
`Fatal error: local() encountered an error (return code 1) while executing 'R CMD BATCH '--args tmp.txt python/out_ut.csv' R/isi_add.r'`